enhancements/proposals/1304-external-seeds/README.md

```diff
@@ -0,0 +1,156 @@
+
+### Non-Goals
+
+- Provide an automated way of creating a multi datacenter ScyllaDB cluster.
+- Provide an automated way of extending existing ScyllaDB clusters with additional datacenters.
+- Support a manual procedure for creating multi datacenter ScyllaDB clusters or extending existing ScyllaDB clusters without connectivity between the datacenters, i.e. this proposal shall not cover the infrastructure setup necessary for it, and only assume it is established.
+
+## Proposal
+
+This document proposes extending the existing ScyllaCluster CR with a new field in its Specification which allows the users to control the external seeds provided to ScyllaDB processes.
+The changes required for ScyllaDB Operator and the sidecar to support the enhancement are also described.
+In addition to that, the document also proposes the procedure for compiling the list of seeds propagated to ScyllaDB processes when external seeds are provided.
+
+### User Stories
+
+#### As a ScyllaDB cluster administrator:
+- I want to control the list of external seeds provided to ScyllaDB nodes.
+- I want to create a multi datacenter ScyllaDB by manually providing the external seeds to chosen datacenters using the ScyllaCluster CR.
+- I want to add a datacenter to existing ScyllaDB datacenters by manually providing the external seeds to chosen datacenters using the ScyllaCluster CR.
+
+### Notes/Constraints/Caveats [Optional]
+
+#### Current role of seeds in ScyllaDB
+
+With [scylladb/scylladb#6848](https://github.com/scylladb/scylladb/pull/6848), the role of seeds in ScyllaDB has been reduced to initial point of contact nodes.
+According to the [documentation](https://opensource.docs.scylladb.com/stable/kb/seed-nodes.html): "**Once the nodes have joined the cluster, seed node has no function**".
+However, investigating the gossiper's code proves that the seed nodes provided to ScyllaDB processes on initialization serve as fallback in case of no live nodes to talk with: https://github.com/scylladb/scylladb/blob/39ca07c49b257d37856164ea5b845f7b98224b98/gms/gossiper.cc#L1013.
+
+Other than that, seeds do not have any additional significance past bootstrap.
+
+#### Multi DC setup procedure
+
+[Create a ScyllaDB Cluster - Multi Data Centers (DC)](https://opensource.docs.scylladb.com/branch-5.3/operating-scylla/procedures/cluster-management/create-cluster-multidc.html#procedure) procedure in ScyllaDB documentation explicitly states to use IP address of the first node, *and only the first node*, as seed for all nodes in all datacenters.
+This has proven to be incorrect in two ways: 
+1. Domain name addresses are also accepted as seeds. 
+2. There is no special significance to the first node of the first DC - it is just a simplification used for the purpose of minimising the risk of creating multiple clusters instead of a single, multi DC cluster.
+
+The specified constrains hold - the datacenters need a matching cluster name and distinct datacenter and rack names.
```

Comment:
Can you clarify the sentence so it's clear whether rack names have to be unique across DC, or just withing a DC?

Reply:
done